suite-native/module-trading/src/components/buy/BuyProviderPicker.tsx

```diff
@@ -2,18 +2,15 @@ import { useSelector } from 'react-redux';
 
 import { BuyTrade } from 'invity-api';
 
-import {
-    TradingRootState,
-    selectBuyQuotesByPaymentMethod,
-    selectTradingBuyIsLoading,
-    selectTradingBuyProviders,
-} from '@suite-common/trading';
+import { selectTradingBuyIsLoading, selectTradingBuyProviders } from '@suite-common/trading';
 import { EventType, analytics } from '@suite-native/analytics';
 import { HStack, Text } from '@suite-native/atoms';
 import { useTranslate } from '@suite-native/intl';
 
 import { useBuyFormContext } from '../../hooks/buy/useBuyFormContext';
 import { useSheetControls } from '../../hooks/general/useSheetControls';
+import { selectMobileBuyQuotesByPaymentMethod } from '../../selectors/buySelectors';
```

Comment:
Can we use native instead to have naming same with the project? Smthng like
```suggestion
import { selectBuyQuotesByPaymentMethodNative } from '../../selectors/buySelectors';
```